akka-docs/src/main/paradox/mailboxes.md

```diff
@@ -63,7 +63,7 @@ then that names a configuration section describing the mailbox type to be
 used.
  2. If the actor’s `Props` contains a mailbox selection—i.e. `withMailbox`
 was called on it—then that names a configuration section describing the
-mailbox type to be used.
+mailbox type to be used (note that this needs to be a top-level section).
```

Comment:
I think "an absolute config path" would be more clear, it does not need to be a top level section AFAIR